test/potluck_test.rb

```diff
@@ -0,0 +1,116 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
```

Comment:
Overall tests look great - but we do want to see a dish_test.rb and potluck_test.rb.  You should have as many test files as you have class files.